src/DossierItem.jsx

```diff
@@ -0,0 +1,16 @@
+import { FlexboxGrid } from 'rsuite';
+
+const DossierItem = ({name, value}) => {
```

Comment:
u can just 

const DossierItem = ({name, value}) => (
   \<Component />
);

without any braces and return

Reply:
makes sense!

Reply:
done